Automattic/wp-calypso
# Nav redesign: delete duplicated controllers

## Proposed Changes

This is a continuation to https://github.com/Automattic/wp-calypso/pull/90009. In this PR, we remove the duplicated controller code from `client/site-monitoring/*` and `client/github-deployments/*`, and use existing `client/my-sites/*` version instead.

## Testing Instructions

1. Open `<Calypso Live URL>/sites?flags=layout/dotcom-nav-redesign-v2`
2. Select a site
3. Smoke-test `Monitoring`, `PHP Logs`, `Server Logs`, and `GitHub Deployments` tab; verify that they behave the same way as Horizon's.

## Pre-merge Checklist
